backend/pkg/api/data_access/vdb_blocks.go

```diff
@@ -65,33 +66,38 @@ func (d *DataAccessService) GetValidatorDashboardBlocks(ctx context.Context, das
 		)
 	if dashboardId.Validators == nil {
 		filteredValidatorsDs = filteredValidatorsDs.
-			From(validators).
+			From(goqu.T("users_val_dashboards_validators").As(validators.GetTable())).
```

Comment:
**thought:** It does not block anything now I just consider the logic here weird.
We defined `validators` above as a table even though no such table exists and then we say it is now just the alias of the actual table `users_val_dashboards_validators`.
I know that it is this way so that `validators` can be used elsewhere as a table but perhaps we can find a better solution in the future.

This is the same for groups but that one also doesn't have a `.GetTable()`.

Reply:
I agree the handling for the different dashboard id types makes it a bit messy and could be improved, but keeping the name as a single definition is still worth it. I've left a code comment :+1: